packages/plugin-lighthouse/src/lib/lighthouse-plugin.ts

```diff
@@ -1,111 +1,77 @@
+import { filterAuditsAndGroupsByOnlyOptions } from './utils';
 
-// No error reporting implemented as in the source Sentry was involved
-/*
-if (cliFlags.enableErrorReporting) {
-  await Sentry.init({
-    url: urlUnderTest,
-    flags: cliFlags,
-    environmentData: {
-      serverName: 'redacted', // prevent sentry from using hostname
-      environment: isDev() ? 'development' : 'production',
-      release: pkg.version,
-    },
-  });
- */
 export function lighthousePlugin(
   url: string,
-  flags?: LighthouseCliFlags,
+  flags?: LighthouseOptions,
 ): PluginConfig {
+  const {
+    skipAudits = [],
+    onlyAudits = [],
+    onlyCategories = [],
+    ...unparsedFlags
+  } = normalizeFlags(flags ?? {});
+
   const { audits, groups } = filterAuditsAndGroupsByOnlyOptions(
-    LIGHTHOUSE_AUDITS,
+    LIGHTHOUSE_NAVIGATION_AUDITS,
     LIGHTHOUSE_GROUPS,
-    flags,
+    { skipAudits, onlyAudits, onlyCategories },
   );
+
+  log(audits, { skipAudits, onlyAudits, onlyCategories });
   return {
     slug: LIGHTHOUSE_PLUGIN_SLUG,
     title: 'Lighthouse',
     icon: 'lighthouse',
     audits,
     groups,
-    runner: createRunnerFunction(url, flags),
+    runner: createRunnerFunction(url, {
+      skipAudits,
+      onlyAudits,
+      onlyCategories,
+      ...unparsedFlags,
+    }),
   };
 }
 
-export function createRunnerFunction(
-  urlUnderTest: string,
-  flags: LighthouseCliFlags = {},
-): RunnerFunction {
-  return async (): Promise<AuditOutputs> => {
-    const {
-      precomputedLanternDataPath,
-      budgetPath,
-      budgets = [],
-      outputPath,
-      ...parsedFlags
-    } = validateFlags({
-      ...DEFAULT_CLI_FLAGS,
-      ...flags,
-    });
-
-    setLogLevel(parsedFlags);
-
-    const config = await getConfig(parsedFlags);
-
-    const budgetsJson = budgetPath ? await getBudgets(budgetPath) : budgets;
-
-    if (outputPath) {
-      await ensureDirectoryExists(dirname(outputPath));
-    }
-
-    const flagsWithDefaults = {
-      ...parsedFlags,
-      budgets: budgetsJson,
-      outputPath,
-    };
-
-    if (precomputedLanternDataPath) {
-      ui().logger.info(
-        `Parsing precomputedLanternDataPath "${precomputedLanternDataPath}" is skipped as not implemented.`,
-      );
-    }
-
-    const runnerResult: unknown = await runLighthouse(
-      urlUnderTest,
-      flagsWithDefaults,
-      config,
+function log(
+  audits: { slug: string }[],
+  opt: { skipAudits: string[]; onlyAudits: string[]; onlyCategories: string[] },
+) {
+  const { skipAudits, onlyAudits, onlyCategories } = opt;
+  const filteredSlugs = new Set(audits.map(({ slug }) => slug));
+  ui().logger.info(
+    `filter: ${JSON.stringify({ skipAudits, onlyAudits, onlyCategories })}`,
+  );
+  ui().logger.info(`audit count: ${JSON.stringify(audits.length)}`);
+  skipAudits.forEach((slug, index) => {
+    ui().logger.info(
+      `Skip audit ${skipAudits.at(
+        index,
+      )} included in audits: ${filteredSlugs.has(slug)}`,
```

Comment:
These logs look nonsensical. 

- there is no context - when running the CLI, it is not at all clear that they come from Lighthouse plugin
  ![image](https://github.com/code-pushup/cli/assets/34691111/f1eaeb38-2a4a-4d67-8d80-76511edd7058)
- logging all included/excluded audits/categories seems excessive, and the message is completely cryptic
  ![image](https://github.com/code-pushup/cli/assets/34691111/adf7b84f-4ad5-434a-91a9-77d918efd254)

I'm not sure any of this information needs to be logged at all, honestly. They look like debug logs to me.

Reply:
> debug logs
I could use verbose? 

What I will change is:
- make them less cryptic
- add verbose flag
- add plugin name to the msg

Reply:
Sounds good :+1: BTW the logger has a `debug` method, maybe that would be a better fit here than `info`.

Reply:
Done